### PR TITLE
eudev -> update & change rootrundir

### PR DESCRIPTION
**Pull request by satmandu**
- `rootrundir` should be `/run`

Works properly:
- [x] x86_64
- [x] armv7l (built with M91 container)
- [x] i686 (@uberhacker a build please?)

### CODE REVIEW

File: packages/eudev.rb [hunk 1]
```diff
@@ -28,16 +26,15 @@ class Eudev < Package
 
   def self.build
     system 'sed -i s,/usr/bin/xsltproc,xsltproc,g man/make.sh'
```
Comment (uberhacker) [inline packages/eudev.rb:28]: Move to `def self.patch` section?

File: packages/eudev.rb [hunk 2]
```diff
@@ -28,16 +26,15 @@ class Eudev < Package
...
-           "--with-rootlibexecdir=#{CREW_PREFIX}/libexec"
+    system 'NOCONFIGURE=1 ./autogen.sh'
+    system "#{CREW_ENV_OPTIONS} ./configure \
+              #{CREW_OPTIONS} \
+              --enable-hwdb \
+              --enable-rule-generator \
+              --with-rootprefix=#{CREW_PREFIX} \
+              --with-rootlibdir=#{CREW_LIB_PREFIX} \
+              --with-rootrundir=/run \
+              --with-rootlibexecdir=#{CREW_PREFIX}/libexec"
```
Comment (uberhacker) [inline packages/eudev.rb:37]: Don't all these options default to the correct locations already?

Comment (satmandu) [reply packages/eudev.rb:37]: You are right.

File: packages/eudev.rb (hunk 1 above)
Comment (satmandu) [reply packages/eudev.rb:28]: done

**Comment by uberhacker** [comment]
Tested on armv7l and i686.  The udevadm executable is not segfaulting.

**Pull request merged**